### DISCUSSION TITLE
Eclipse IDE error, unable to install J2ME

### DISCUSSION THREAD

**Discussion by hieujunior452** [eclipse-platform/eclipse.platform#1269] [Q&A]
I am getting the error below when installing j2me plugins for Eclipse I don't know how to fix it, can someone help me?
Plugins: oracle-jmesdk-8-3-rr-eclipse-plugins.zip
The operation cannot be completed. See the details.Cannot complete the installation because one or more required items could not be found.
Software being installed: Java ME SDK Tools 8.3.0.201604172311 (com.oracle.jme.toolkit.feature.group 8.3.0.201604172311)
Missing requirement: Java ME SDK Tools 8.3.0.201604172311 (com.oracle.jme.toolkit.feature.group 8.3.0.201604172311) requires 'org.eclipse.equinox.p2.iu; org.eclipse.mtj.feature.group 0.0.0' but it could not be found
![image](https://github.com/eclipse-platform/eclipse.platform/assets/111551139/8bfd5e47-2878-44f6-9bf9-978de6810271)

**Comment by akurtakov** [comment]
This is not Eclipse Platform issue. And the missing project https://projects.eclipse.org/projects/tools.mtj has been archived for some reason. I recommend trying to contact people that were involved into it to figure out what happened with it. 
There is nothing that could be done by Eclipse Platform thus closing.
